utils/validate-facility-ids/database-client/mongo-db-client.ts

```diff
@@ -0,0 +1,17 @@
+import z from 'zod';
+import { MongoDbClient } from '@ukef/dtfs2-common/mongo-db-client';
+
+const mongoDbConfigSchema = z.object({
+  MONGODB_URI_QA: z.string(),
+  MONGO_INITDB_DATABASE: z.string(),
```

Comment:
Could we add in the README section that these need to be set to the values for the env you want to run against?